src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -352,7 +350,7 @@ public boolean isProtected() {
     /**
      * Writes all data in the format <key, serialized data>.
      */
-    public Map<String, String> serialize() {
+    public Map<String, String> getAsString() {
```

Comment:
I think that makes sense. Can you change that please @tobiasdiez and merge it in? :trophy: